requirements-dev.txt

```diff
@@ -25,17 +26,19 @@ cryptography==2.1.3
 docker==2.6.1
 docker-pycreds==0.2.1
 docutils==0.14
-domovoi==1.4.5
+domovoi==1.5.1
```

Comment:
Please rebase this on master - I committed a change that conflicts with this.

Reply:
done